content/blog/2016-07-23-pull-does-not-scale-or-does-it.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Pull doesn't scale - or does it?
+created_at: 2016-07-23
+kind: article
+author_name: Julius Volz
+---
+
+Let's talk about one particularly hard-to-kill myth again. Whenever there is a
+discussion about monitoring systems and Prometheus's pull-based metrics
+collection approach comes up, someone inevitably chimes in about how a
+pull-based approach just “fundamentally doesn't scale”. When asked why they
+think this is the case, they either respond with vague fears or cite reasons
```

Comment:
That's an example for the lecturing wording I meant by the way. If you want to convince proponents of the push model, I'd suggest to use a less alienating wording.